bin/raptor

```diff
@@ -223,17 +233,25 @@ parent
     default: '',
     transform: utils.fromEnvironment('RAPTOR_DATABASE', '')
   })
-  .option('log', {
+  .option('metrics', {
     metavar: '<path>',
-    help: 'path to store historical test metrics. Environment: RAPTOR_LOG',
-    default: defaultLog,
-    transform: utils.compose(utils.toAbsolute, utils.fromEnvironment('RAPTOR_LOG', defaultLog))
+    help: 'path to store historical test metrics. Environment: RAPTOR_METRICS',
+    default: defaultMetricsFile,
+    transform: utils.compose(utils.toAbsolute, utils.fromEnvironment('RAPTOR_METRICS', defaultMetricsFile))
   })
   .option('output', {
     metavar: '<mode>',
-    help: 'output mode: normal or quiet. Environment: RAPTOR_OUTPUT',
-    default: 'normal',
-    transform: utils.fromEnvironment('RAPTOR_OUTPUT', 'normal')
+    help: 'console output mode. Environment: RAPTOR_OUTPUT',
+    choices: ['console', 'json', 'quiet'],
+    default: 'console',
+    transform: utils.fromEnvironment('RAPTOR_OUTPUT', 'console')
+  })
+  .option('batch', {
+    metavar: '<count>',
+    help: 'batch database requests to <count> number of records',
```

Comment:
What does this option actually do? I see further down it is an option when creating the db client... does that mean it will only actually write the db in batches of <count> records?